Index: app/src/main/java/com/example/androidprojectcollection/MatchGameActivity.java
===================================================================
diff --git a/app/src/main/java/com/example/androidprojectcollection/MatchGameActivity.java b/app/src/main/java/com/example/androidprojectcollection/MatchGameActivity.java
deleted file mode 100644
--- a/app/src/main/java/com/example/androidprojectcollection/MatchGameActivity.java	(revision 1965f35a85911900d6f08086f18b476695ebb8a2)
+++ /dev/null	(revision 1965f35a85911900d6f08086f18b476695ebb8a2)
@@ -1,15 +0,0 @@
-package com.example.androidprojectcollection;
-
-import androidx.appcompat.app.AppCompatActivity;
-
-import android.os.Bundle;
-
-public class MatchGameActivity extends AppCompatActivity {
-
-    @Override
-    protected void onCreate(Bundle savedInstanceState) {
-        super.onCreate(savedInstanceState);
-        setContentView(R.layout.activity_match_game);
-
-    }
-}
\ No newline at end of file
Index: app/src/main/res/layout/activity_match_game.xml
===================================================================
diff --git a/app/src/main/res/layout/activity_match_game.xml b/app/src/main/res/layout/activity_match_game.xml
deleted file mode 100644
--- a/app/src/main/res/layout/activity_match_game.xml	(revision 1965f35a85911900d6f08086f18b476695ebb8a2)
+++ /dev/null	(revision 1965f35a85911900d6f08086f18b476695ebb8a2)
@@ -1,9 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
-<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
-    xmlns:app="http://schemas.android.com/apk/res-auto"
-    xmlns:tools="http://schemas.android.com/tools"
-    android:layout_width="match_parent"
-    android:layout_height="match_parent"
-    tools:context=".MatchGameActivity">
-
-</androidx.constraintlayout.widget.ConstraintLayout>
\ No newline at end of file
